docs/proxy-config.md

```diff
@@ -105,3 +106,20 @@ OPTIONS
                                   'production' (default: sandbox)
     -o --output=<value>           Output format. One of: json|yaml
 ```
+
+### Deploy
+
+```shell
+NAME
+    deploy - Promotes the APIcast configuration to the Staging Environment
```

Comment:
Why has it been done as a standalone command and not part of the `promote` command?

Reply:
promote command, as it says in its description, promotes latest staging conf to production env
```
3scale proxy-config promote -h
NAME
    promote - Promote latest staging Proxy Configuration to the production environment
```

Reply:
That's the current description.

Why has not been placed for example as a flag of the promote command, for example `--environment <env>`?

Reply:
It would imply full refactoring of the command, and still will not be very clear. For instance, check `promotable?` method being used by the current command. That will not make sense. 

I think it is cleaner to have its own command instead of being part of another command using optional input param

Reply:
:+1: